src/settings/MappingProfiles/detailsSections/view/OrderDetailsSection/FundDistribution.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  Accordion,
+  Row,
+  Col,
+  NoValue,
+} from '@folio/stripes/components';
+
+import { ViewRepeatableField } from '../ViewRepeatableField';
+
+import {
+  getFieldValue,
+  renderAmountValue,
+  transformSubfieldsData,
+} from '../../utils';
+import { TRANSLATION_ID_PREFIX } from '../../constants';
+import { mappingProfileFieldShape } from '../../../../../utils';
+
+export const FundDistribution = ({ mappingDetails }) => {
+  const noValueElement = <NoValue />;
+
+  const fundDistributions = getFieldValue(mappingDetails, 'fundDistribution', 'subfields');
+
+  const fundDistributionsVisibleColumns = ['fundId', 'expenseClassId', 'value'];
```

Comment:
Could you please investigate possible to move  `'fundId', 'expenseClassId', 'value' `to const and reuse it in this case and in similar cases across all pull?